### PR TITLE
Ex. 4: Like Counter

### PR DESCRIPTION
**Pull request by sfayson**

### CODE REVIEW

File: src/Student10/Ex4/Tweet.js [hunk 1]
```diff
@@ -0,0 +1,13 @@
+import React from 'react'
+
+
+ 
+export const Tweet = (props) => {
+        
+  return (
+    <div className='Tweet' style={{ backgroundColor: props.color }}>
+        {props.message}
+        <button onClick={() => props.onHandleClick(props.message * 2)}> Like</button>
```
Comment (mburolla) [inline src/Student10/Ex4/Tweet.js]: There's no reason to pass `props.message * 2` into this callback, and it doesn't make sense.  Why would we multiply the message times two?

File: src/Student10/Ex4/Tweets.js [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+    setCounter(++counter);
+  };
+
+  return (
+    <div className="Tweets">
+      Total Likes: {counter}
+      <Tweet
+        message="It's payday!!!"
+        color="green"
+        onHandleClick={(n) => onHandleClick(n)}
```
Comment (mburolla) [inline src/Student10/Ex4/Tweets.js]: No reason to pass n in and out of this function.

File: src/Student10/Ex4/Tweets.js [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+      />
+      <Tweet
+        message="I just bought a new car!"
+        color="yellow"
+        onHandleClick={(n) => onHandleClick(n)}
```
Comment (mburolla) [inline src/Student10/Ex4/Tweets.js]: No reason to pass n in and out of this function.

File: src/Student10/Ex4/Tweets.js [hunk 4]
```diff
@@ -0,0 +1,31 @@
...
+      />
+      <Tweet
+        message="I just smashed my new car."
+        color="red"
+        onHandleClick={(n) => onHandleClick(n)}
```
Comment (mburolla) [inline src/Student10/Ex4/Tweets.js]: No reason to pass n in and out of this function.

File: src/Student10/Ex4/Tweet.js (hunk 1 above)
Comment (sfayson) [reply src/Student10/Ex4/Tweet.js]: Removed it

**Pull request merged**